src/dso_api/settings.py

```diff
@@ -105,6 +105,7 @@
 
 ROOT_URLCONF = "dso_api.urls"
 
+STATICFILES_DIRS = [str(BASE_DIR / "dso_api/static")]
```

Comment:
```suggestion
STATICFILES_DIRS = [str(BASE_DIR / "dso_api" / "static")]
```

But shouldn't we call this `MEDIA_ROOT` or `STATIC_ROOT` to conform to [Django conventions](https://docs.djangoproject.com/en/3.2/howto/static-files/)?

Reply:
Oh, we already have `STATIC_ROOT` and we also have `django.contrib.staticfiles` in the `INSTALLED_APPS`. Any reason not to use that for these new files?

Reply:
@lbam The `STATIC_ROOT` is where all collected files get *written* to. So you can't use that for providing assets!

Reply:
Oh, my bad, misread the docs. Never mind!